app/views/admin/communities/footer/edit.haml

```diff
@@ -34,7 +34,7 @@
     %h3= t("admin.communities.footer.footer_style")
     .row
       .col-12.footer-theme-radio-buttons
-        = form.collection_radio_buttons :footer_theme, @service.foter_themes, :key, :value
+        = form.collection_radio_buttons :footer_theme, @service.footer_themes, :key, :value
```

Comment:
Please create sane patches! The single patch must implement a single feature and do not break anything.

Reply:
@aivils My understanding is that this was done through another PR, at https://github.com/sharetribe/sharetribe/pull/3749.

It definitely would be better to have everything in a single PR, and if @landsverk can do that it would be perfect. But maybe we could also accept both PR at the same time, to still welcome the contribution?

@aivils I trust how you prefer to do in regards to best practices.
@landsverk Ideally you should apply both changes into that single PR and close the other one.